cirq/linalg/decompositions.py

```diff
@@ -854,3 +857,98 @@ def kak_decomposition(unitary_object: Union[np.ndarray, 'cirq.SupportsUnitary'],
         global_phase=g * inner_cannon.global_phase,
         single_qubit_operations_before=(b1, b0),
         single_qubit_operations_after=(a1, a0))
+
+
+def kak_vector(unitary: np.ndarray):
```

Comment:
Add a `kak_vector` entry in `api.rst` right after `kak_decomposition`, so that it gets included in documentation. Rendered output looks like this currently (note sigma spelled instead of symbol):

![image](https://user-images.githubusercontent.com/79941/66352191-f8d6fc00-e913-11e9-81cb-16ceec01c6bc.png)

Reply:
How do I check the rendered output?

Reply:
Build the docs locally by running https://github.com/quantumlib/Cirq/blob/master/dev_tools/build-docs.sh.

Reply:
Currently there isn't a fast convenient way other than to invoke `./dev_tools/build-docs.sh` from the repo root.

Reply:
[This](https://screenshot.googleplex.com/zoupWMO1opG) is what I get when I try to build. I previously ran `pip install pypandoc` and it seems to have installed.

Reply:
Try `pip install -r dev_tools/conf/pip-list-dev-tools.txt`

Reply:
That seems to fix it, thanks! The docstring seems to build correctly now.